### PR TITLE
More cleanup

### PR DESCRIPTION
**Pull request by aslagle**
Some additional cleanup and class-level comments for many of the components.

### CODE REVIEW

**Comment by aslagle** [comment]
@EdwinGuzman let me know if these comments are helpful or if there's something better I could be doing.

**Comment by EdwinGuzman** [comment]
Looks good on a first glance. Are you following a specific documentation syntax, like esdoc?

**Comment by aslagle** [comment]
I was trying to be compatible with http://typedoc.org/ but the comments don't have much detail.

**Comment by EdwinGuzman** [comment]
Thanks, I'll looking into that.

File: packages/opds-web-client/src/AuthPlugin.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import { AuthFormProps, AuthButtonProps } from "./components/AuthProviderSelectionForm";
 import { AuthMethod, AuthCredentials } from "./interfaces";
 
+/** Applications can implement this interface if they would like to support authentication
+    methods other than basic auth. A list of `AuthPlugin`s can be passed as a prop to the
```
Comment (EdwinGuzman) [inline packages/opds-web-client/src/AuthPlugin.ts]: Don't you need to start new lines with `*` and start the comment after the first line? Unless if this gets read the same way?

Comment (aslagle) [reply packages/opds-web-client/src/AuthPlugin.ts]: I haven't actually tried running this through typedoc, but it's a valid comment. I'm not sure.

**Pull request merged**